docs/api/command-line-switches.md

```diff
@@ -16,78 +16,64 @@ app.on('ready', () => {
 })
 ```
 
-## --ignore-connections-limit=`domains`
```

Comment:
I organized these flags to group Electron-specific flags separately from Node.js flags and also alphabetize them.